### ISSUE TITLE
[Accessibility ♿] - Missing Tooltip for Add Revision Button

### ISSUE THREAD

**Issue by tobidosumu** [mozilla/perfcompare#423]
### Issue Description

The `AddRevisionButton` component in the application lacks a tooltip. This means that users who rely on screen readers or have other visual impairments may not be able to understand the purpose of the button. A tooltip should be added to provide a description of the button's functionality.

### Steps to Reproduce

1. Open the application.

2. Locate the Add Revision button.

3. Hover over the button, or attempt to click on it.

### Expected Behavior

When a user hovers over the Add Revision button, a tooltip should be displayed that provides a description of the button's functionality.

### Actual Behavior

When a user hovers over the Add Revision button, no tooltip is displayed.
![addBtnTooltip-before](https://user-images.githubusercontent.com/60399800/226222005-fae1c87f-0460-448f-ba0c-5c227b712629.gif)

@Carla-Moz @kimberlythegeek

**Comment by siddhi-244** [comment]
Can i work on this? @kimberlythegeek @Carla-Moz
